XPU and MPS take 3 (#276)
Cambridge-ICCS/FTorch: examples/3_MultiGPU/README.md

To run this example requires:

- CMake

> - Two (or more) CUDA or XPU GPU devices (or a single Mps device).

Author: Do we want this, for consistency?
```suggestion
- Two (or more) CUDA or XPU GPU devices (or a single MPS device).
```

and should match the compiler that was used to locally build FTorch.)

To run the compiled code calling the saved `SimpleNet` TorchScript from

> Fortran, run the executable with argumentsof device type and the saved model file:

Author: ```suggestion
Fortran, run the executable with arguments of device type and the saved model file:
```

output on device 0: [  0.0,  2.0,  4.0,  6.0,  8.0]
output on device 1: [  2.0,  4.0,  6.0,  8.0, 10.0]
```

> Again, note that Mps will only use device 0.

Author: Again?
```suggestion
Again, note that MPS will only use device 0.
```
